src/backend/distributed/citus--7.5-5--7.5-6.sql

```diff
@@ -0,0 +1,35 @@
+/* citus--7.5-1--7.5-2 */
+SET search_path = 'pg_catalog';
+
+CREATE FUNCTION pg_catalog.poolinfo_valid(text)
+	RETURNS boolean
+    LANGUAGE C STRICT
+    AS 'MODULE_PATHNAME', $$poolinfo_valid$$;
+COMMENT ON FUNCTION poolinfo_valid(text) IS 'returns whether a poolinfo is valid';
```

Comment:
Super minor, feel free to ignore: it's a little odd that we're schema-qualifying the `CREATE FUNCTION`, but not in the `COMMENT ON FUNCTION`

Reply:
This is largely copy-pasted from other upgrade scripts so not gonna bother to change (to avoid the Travis round-trip).